### DISCUSSION TITLE
Multiple Custom Feeds

### DISCUSSION THREAD

**Discussion by code-a-man** [dailydotdev/daily#719] [Feature Requests]
I am following so many tags from different titles and it will be cool if we can have multiple feeds separated by topics.
For example, if I want to look for blockchain related posts only there should be a section under "My Feed" named "Blockchain" created by me.
It can go like that:
Create Custom Feed -> Name: "Blockchain" (Custom name by user) -> Select tags for this feed -> Done!
Now my left bar should look like this:
**Feeds**
- My Feed
- Blockchain

**Discover**
- ...

**Comment by idoshamun** [comment]
Hi @code-a-man,

Thanks for the suggestion. Let's keep the discussion here: https://github.com/dailydotdev/daily/issues/336
